src/plugins/simulator/fs9/simulatorfs9.cpp

```diff
@@ -194,6 +194,15 @@ namespace BlackSimPlugin::Fs9
         return CSimulatorFsCommon::physicallyRemoveRemoteAircraft(callsign);
     }
 
+    bool CSimulatorFs9::dontOverwrite(const CFrequency &from, const CFrequency &to)
+    {
+        bool ergeb = false;
```

Comment:
Avoid german abbreviations inside the code

Reply:
this change can be removed again